src/modules/wallet.js

```diff
@@ -0,0 +1,50 @@
+import {WalletRpc} from '../client/wallet-rpc';
+
+/** Wallet */
+export class Wallet {
+  /**
+   * Load wallet
+   */
+  async load() {
+    await WalletRpc.create('dock-wallet');
```

Comment:
any danger if this gets called twice?

Reply:
I've updated this code, it will avoid duplicated calls to load.